package.json

```diff
@@ -39,4 +41,4 @@
   "tap": {
     "check-coverage": true
```

Comment:
And in fact, in this repo, the `npm test` fails if it's not 100% covered, because of this line.